src/xncml/generated/ncml_2_2.py

```diff
@@ -0,0 +1,862 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Type
+
+__NAMESPACE__ = "http://www.unidata.ucar.edu/namespaces/netcdf/ncml-2.2"
```

Comment:
Any reason this variable is not reused? The same URI is hardcoded many places below.

Reply:
Not sure. In all honesty, I've never developed this library; I'm just coming in to apply the boilerplate code template. I'm more concerned that the URL doesn't resolve.

Reply:
I guess it is something unidata would have to fix on their website. 
I can find the same URI in archives (https://fossies.org/linux/nco/data/ncml-2.2.xsd).